### DISCUSSION TITLE
Error in nlmixr2 models

### DISCUSSION THREAD

**Discussion by ksadko** [nlmixr2/rxode2#484] [Q&A]
Hi, 
   My lab recently got a new computer and after trying to run previous models made with nlmixr2 or even the sample models on the website this error occurs. 
![Screenshot 2023-03-22 124355](https://user-images.githubusercontent.com/128625747/226976901-20d2f5d1-7685-4f37-9eac-65f1b987c3be.png)
Is there anything that can be done to fix this or could this be tied to the RStudio update? This code runs just fine on other computers.

**Comment by billdenney** [comment]
We are currently in the process of updating many packages on CRAN.  (See #479)  Due to the timing of CRAN approving the packages, they are not in a consistent state right now.  The simplest solution right now is to install from either R-universe or GitHub (https://github.com/nlmixr2/rxode2#installation).

**Reply by mattfidler** [reply]
The R-universe install instructions for `nlmixr2` is here:

https://github.com/nlmixr2/nlmixr2#install-using-the-r-universe
